src/services/schedule.ts

```diff
@@ -15,15 +15,18 @@ export interface MessageInfo {
 }
 
 export class Scheduler {
-  private client: Client;
+  jobStore: Map<string, Job>;
```

Comment:
You can initialise a property in the definition without doing so in the constructor.

```ts
export class Scheduler {
  jobStore: Map<string, Job> = new Map();
  scheduleJob = scheduleJob
  constructor(private client: Client) {}
```